contracts/tokens/ERC20HoldableToken.sol

```diff
@@ -20,32 +20,60 @@ import "./IERC20HoldableToken.sol";
 contract ERC20HoldableToken is ERC20Token, IERC20HoldableToken {
     using SafeMath for uint256;
 
-    struct HoldData {
-        address sender;
-        address recipient;
-        address notary;
-        uint256 amount;
-        uint256 expirationDateTime;
-        bytes32 lockHash;
-        HoldStatusCode status;
-    }
-
     // mapping of accounts to hold data
-    mapping(bytes32 => HoldData) internal holds;
+    mapping(bytes32 => ERC20HoldData) internal holds;
     // mapping of accounts and their total amount on hold
     mapping(address => uint256) internal accountHoldBalances;
+
+    mapping(bytes32 => bytes32) internal _holdHashToId;
+
     uint256 override public totalSupplyOnHold;
 
     modifier isHeld(bytes32 holdId) {
         require(
-            holds[holdId].status == HoldStatusCode.Held,
-            "Hold is not in Held status"
+            holds[holdId].status == HoldStatusCode.Ordered,
```

Comment:
Status can also be "ExecutedAndKeptOpen" here (like we have in "_holdCanBeExecuted" function in ERC1400TokensValidator)

Reply:
should there be an option to execute a hold and keep it open in `ERC20HoldableToken` ? Currently there is no way to do so. The same `HoldStatusCode` enum is used between the two tokens for compatibility and contract size purposes

Reply:
should there be an option to execute a hold and keep it open in ERC20HoldableToken ? ==> I'd say not as part of this MR but in target yes as in target we shall use the same code to manage holds for ERC20s and ERC1400s if possible.